templates/zerver/reset_confirm.html

```diff
@@ -33,6 +33,7 @@ <h1>{{ _('Set a new password.') }}</h1>
                       value="{% if form.new_password1.value() %}{{ form.new_password1.value() }}{% endif %}"
                       maxlength="100"
                       data-min-length="{{password_min_length}}"
+                      data-max-length="100"
```

Comment:
I'd like to double check if there is a reason why `maxlength` hard codes `100` here instead of using the global setting variable. I currently follow the file's previous convention but feel that the hard coding might just be due to global setup differences back then.

Reply:
This seems like an error in the previous PR. We always use variables for this sort of thing where possible, and it seems very possible. Might need to just add the variable to the template rendering context.